doc/swish/erlang.tex

```diff
@@ -2979,6 +2979,29 @@ \subsection{String Utilities}
 \var{separator}. When \var{last-separator} is specified, it is used as
 the last separator.
 
+\pagebreak
```

Comment:
This manual page break may introduce a maintenance burden.

Reply:
Good catch. I left that in by mistake.